lib/experimental/new_embed.dart

```diff
@@ -48,6 +48,7 @@ class NewEmbed {
   DisableableButton reloadGistButton;
   DisableableButton formatButton;
   DisableableButton showHintButton;
+  DisableableButton moreButton;
```

Comment:
nit: "menuButton" might be a more standard way of referring to a hamburger icon, but feel free to push back.